src/workspace-data/Data.js

```diff
@@ -1496,6 +1496,7 @@ export const WorkspaceData = _.flow(
                       dataProvider: wdsDataTableProvider,
                       recordType: selectedData.entityType,
                       wdsSchema: wdsTypes.state,
+                      loadMetadata,
```

Comment:
This `loadMetadata` won't do anything for WDS tables. WDSContent shadows `entityMetadata` with its own state variable. Thus, WDSContent would need to pass DataTable a `loadMetadata` function that updates WDSContent's state variable.

Reply:
Yeah WDS doesn't seem to have the equivalent of `entityMetadata`?  It doesn't set anything in `azureLoadEntityMetadata`.  `loadMetadata` should call `loadWdsSchema` but that apparently doesn't result in a refresh?

Reply:
WDSContent translates the WDS schema into something that looks like entity metadata.

Reply:
Ah. It reloads the schema in Data. But WDSContent has stashed the derived entity metadata in a state variable, which is never updated when the schema passed down from Data changes. WDSContent ignores all changes to schema.